linera-explorer/src/lib.rs

```diff
@@ -202,6 +224,117 @@ async fn applications(node: &str, chain_id: &ChainId) -> Result<(Page, String),
     ))
 }
 
+fn list_entrypoints(types: &[Value], name: &Value) -> Option<Value> {
+    types
+        .iter()
+        .find(|x: &&Value| &x["name"] == name)
+        .map(|x| x["fields"].clone())
+}
+
+fn fill_type(t: &Value, types: &Vec<Value>) -> Value {
```

Comment:
sorry... it is kind of generic function. The introspection query gives the list of queries, the list of mutations, the list of subscriptions and a list of types used in all the former objects. I fill these objects with the types stored in this types library. Since I add some fields (`_input`, `_include`) to these objects, I cannot use a reference to this types library